src/unstable/components/Form/widgets/PasswordWidget.tsx

```diff
@@ -39,7 +44,8 @@ export interface PasswordStrengthProps {
 }
 
 const PasswordStrength = ({ password }: PasswordStrengthProps) => {
-	const [strengthScore, setStrengthScore] = React.useState();
```

Comment:
I don't think this change is necessary to fix the typings, let's keep it as it was.

Reply:
the problem was in the typing of the strengthScore, since it starts with an undefined value and as explained in the link attached in the PR, undefined cannot be used as index type.
The other solution that we can apply is to set strengthScore to NaN as a starting value, but it seemed like a risky workaround.
Were you thinking of another way?

https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types

Reply:
@JSReds From what I can see it can also start as `0`

Reply:
@sradevski totally agree but starting with 0 will skip the `|| strengthScore === undefined` check. Should we change the control or delete it and always render the component with 0 width ?

maybe I would opt to change the rule to `!strengthScore` and delete index 0 `{width: 0}`, from `STRENGTH_STYLE` as it will never be used. what do you think about it  ?